### ISSUE TITLE
Switch network, list sometimes takes more time to load, can't select different network until loaded

### ISSUE THREAD

**Issue by 314159265359879** [blockstack/blockstack-app#5453]
Steps to reproduce
1. Open wallet with 50 or so accounts
2. Switch account scroll over, the accounts let them load
3. Then when the wallet is throttling 
4. Click top right corner menu
5. Switch network
6. See issue:
network isn't clickable until the clouds disappear 

https://github.com/leather-wallet/extension/assets/33360391/bf2a64d8-408b-4c7c-b828-e3fa461c86a3

Users don't expect to have to wait for this. Is this related to the wallet throttling? I do not see any connection issues in the console when selecting "switch networks". 

On a fresh wallet, with one account, clicking switch network, loads them very quickly less then a second before the clouds go away and they are selectable.

**Comment by markmhendrickson** [comment]
@alter-eggo any idea if this is related to client-side throttling?

**Comment by 314159265359879** [comment]
This can also occur when connecting to a dapp, likely related: https://github.com/leather-wallet/extension/issues/5509

**Comment by alter-eggo** [comment]
@markmhendrickson yes, this is throttling problem, need to put higher priority for request that check network availability

**Comment by 314159265359879** [comment]
With the throttling improvements in https://github.com/leather-io/extension/pull/5636 it is still possible you have to wait up to 1 minute for the network to be clickable.

**Comment by markmhendrickson** [comment]
@alter-eggo is there addition work we can do for this issue to make the selection of a network immediate, even if other queries are still pending resolution?

**Comment by alter-eggo** [comment]
@markmhendrickson I'll check
